common/core/hwclayer.cpp

```diff
@@ -223,6 +223,12 @@ int32_t HwcLayer::GetAcquireFence() {
 
 void HwcLayer::SufaceDamageTransfrom() {
   int ox = 0, oy = 0;
+
+  if (surface_damage_ == display_frame_) {
```

Comment:
I believe this may result in some bugs?
Suppose a source crop is larger than display_frame_ case,
partial damage can result in surface_damage equal to display_frame?
But it still need transform

Reply:
OK, I will add a status in SetSurfaceDamage to avoid transform if no damage is sent by invokeing this method.